caffe2/operators/relu_op.cc

```diff
@@ -29,8 +29,9 @@ bool ReluOp<float, CPUContext>::RunOnDevice() {
 
 template <>
 bool ReluGradientOp<float, CPUContext>::RunOnDevice() {
-  auto& Y = Input(0);
-  auto& dY = Input(1);
+  auto& X = Input(0);
```

Comment:
Relu gradient needs only Y and dY also the variable X isn't  used in the code section bellow :)